addons/sale_loyalty_delivery/tests/test_free_shipping_reward.py

```diff
@@ -315,9 +315,9 @@ def test_free_shipping_reward_last_line(self):
         claimable_rewards = order._get_claimable_rewards()
         self.assertEqual(len(claimable_rewards), 1)
         # Try to apply the loyalty card to the sale order
-        self._apply_promo_code(order, loyalty_card.code)
+        self.assertTrue(self._claim_reward(order, loyalty_program))
```

Comment:
Because I realized that I understood the function frong, and by default if there is no code provided it looks through the program to find if there is a loyalty card.